src-docs/src/services/playground/props.js

```diff
@@ -23,10 +23,7 @@ const getProp = (prop, propName) => {
       newProp.required = prop.required;
       if (prop.defaultValue) {
         newProp.defaultValue = prop.defaultValue.value;
-        newProp.value = prop.defaultValue.value.substring(
-          1,
-          prop.defaultValue.value.length - 1
-        );
+        newProp.value = prop.defaultValue.value;
```

Comment:
![Screenshot 2020-07-14 at 12 01 18 AM](https://user-images.githubusercontent.com/43617894/87340028-27e66200-c565-11ea-83d5-8348d83b91f9.png)

Some times `defaultValue` is `null`

so, 
```
        newProp.value = prop.defaultValue && prop.defaultValue.value;
```